spec.md

```diff
@@ -234,8 +234,6 @@ When using compound namespaces, there MUST NOT be more than two sub-namespaces w
 That is, the following is allowed:
 
 ```php
-<?php
-
```

Comment:
I'd leave this one.  We're talking about the `use` statements, which happen at a very specific place in the file, so I think it's fine to leave here.